sdk/attestation/azure-security-attestation/test/ut/tpmattestation_test.cpp

```diff
@@ -25,6 +25,9 @@ namespace Azure { namespace Security { namespace Attestation { namespace Test {
 
   // cspell: words aikcert
   class TpmAttestationTests : public Azure::Core::Test::TestBase {
+  public:
+    TpmAttestationTests() { TestBase::SetUpTestSuiteLocal(AZURE_TEST_ASSETS_DIR); };
```

Comment:
Why is this only needed for TPM attestation and not all the other test classes? Is there a problem if TPM attestation isn't set up?

Reply:
i couldn't figure out where the other base classes are in attestation. To enable it we woudl need to make the same changes to attestation